README.md

```diff
@@ -0,0 +1,53 @@
+# Flatpak for DOSBox
+
+## Installation
+
+This Flatpak is available on
+[Flathub](https://flathub.org/apps/details/com.dosbox.DOSBox).
+After following the [Flatpak setup guide](https://flatpak.org/setup/),
+you can install it by entering the following command in a terminal:
+
+```bash
+flatpak install --user flathub com.dosbox.DOSBox -y
+```
+
+Once the Flatpak is installed, you can run DOSBox using your desktop environment's
+application launcher.
+
+## Updating
+
+This Flatpak follows the latest stable DOSBox version.
+To update it, run the following command in a terminal:
+
+```bash
+flatpak update
+```
+
+## Limitations
+
+- None known. Please report an issue if you find any.
+
+## Configuration files
+
+Under the default Flatpak configuration, the DOSBox configuration files are
+located in `$HOME/.var/app/com.dosbox.DOSBox/.dosbox/`. To access it with a
+graphical file manager, you'll have to make hidden folders visible.
+
+## Building from source
+
+Install Git, follow the
+[flatpak-builder setup guide](https://docs.flatpak.org/en/latest/first-build.html)
+then enter the following commands in a terminal:
+
+```bash
+git clone --recursive https://github.com/flathub/com.dosbox.DOSBox.git
+cd com.dosbox.DOSBox
+flatpak install --user flathub org.freedesktop.Sdk//19.08 -y
```

Comment:
This will work only if there is an `--user` flathub repository already added. Default is `--system`.